### DISCUSSION TITLE
How to omit an attribute when saving a Lucid model?

### DISCUSSION THREAD

**Discussion by juliardi** [adonisjs/core#2343] [Help] [answered]
Hi, I am new to AdonisJS, and right now I am trying to make a simple API. I add a `raw_password` attribute to a User model, but I don't want it to be saved to the database. Because Lucid automatically generates a query, that `raw_password` field is always inserted into the query and resulted in an error from the database.
My question is, how to omit an attribute when saving a Lucid model?

**Comment by RomainLanz** [comment, marked as answer]
Hey @juliardi! 👋 

What version of the framework are you using?

**Reply by juliardi** [reply]
I am using version 4.1.0

**Reply by RomainLanz** [reply]
Then the easiest thing you can do is use the `delete` operator before saving.

```js
delete user.raw_password

await user.save()
```

You can find an example in our Blog Demo.
➡️ https://github.com/AdonisCommunity/adonis-blog-demo/blob/master/app/Controllers/Http/UserController.js#L49

**Reply by juliardi** [reply]
Thank you for your suggestion
